### DISCUSSION TITLE
How to install fpp by local file

### DISCUSSION THREAD

**Discussion by Zanejins** [nasa/fprime#1958] [Q&A] [answered]
I can't access github on my ubuntu due to network problems.
But I have downloaded native-fpp-linux-x86_64 and fprime-fpp-1.0.2.

When I run python setup.py in the fprime-fpp-1.0.2 directory, it still be from github, how do I change it to native-fpp-linux-x86_64 which I have downloaded locally. Thank you.
![230534976-8bb150ec-7580-492c-a92f-2ecf1248e95a](https://user-images.githubusercontent.com/98217021/230553893-93bfeb19-5f7a-4548-8ae8-bba2ab4f0b39.png)

**Comment by ThibFrgsGmz** [comment, marked as answer]
If I remember well, you can hack the installation by changing the various paths in `fprime_fpp_install.py`.

https://github.com/fprime-community/fpp/blob/f8290b9f9c22f663b8922df8845da7f739941cb8/fprime_fpp/fprime_fpp_install.py#L78

I did it once a few months ago because like you I had no internet connection.

**Comment by Zanejins** [comment]
yes, I solve this problem with your suggestions. Thank you
